src/components/Spinner.js

```diff
@@ -38,7 +38,12 @@ class Spinner extends React.Component {
 
     return (
       <Animated.Image
-        style={{ height: this.props.size, width: this.props.size, transform: [{ rotate: spin }] }}
+        style={[
+          this.props.style, {
```

Comment:
Done in af1ef6c4a0c01af7fd8302caa2d2430cf5783107

Had to ignore an eslint error because it does not allow us to accept objects

We could describe the object we expect, but we would have to specify exactly what styles we accept, it would look something like:

```typescript
Spinner.propTypes = {
  style: PropTypes.shape({
      marginLeft: PropTypes.number,
      marginRight: PropTypes.number,
      paddingLeft: PropTypes.number,
      paddingRight: PropTypes.number,
      ...
    })
};
```